pom.xml

```diff
@@ -11,6 +11,9 @@
   <name>Bytecode transformation-based library for managing backward compatibility</name>
   <version>1.6-SNAPSHOT</version>
 
+  <properties>
+    <maven-release-plugin.version>2.5.2</maven-release-plugin.version>
```

Comment:
Ugh! I *hate* that property stuff from the jenkins parent... such an anti-pattern... many dead kittens